src/server/helpers/config.js

```diff
@@ -9,7 +9,8 @@ import dotenv from 'dotenv';
  */
 const CLIENT_SIDE_WHITELIST = [
   'NODE_ENV',
-  'BASE_URL'
+  'BASE_URL',
+  'LP_API_USERNAME'
```

Comment:
I've added a script to create Launchpad credentials, referred to it from `docs/environment-variables.md`, and removed this feature switch.